### PR TITLE
Python26 Python27 and Python34 compat

### PR DESCRIPTION
**Pull request by Natim**
Based on #9.

### CODE REVIEW

**Comment by Natim** [comment]
@jvehent According to your tests it should work.

Feel free to try in a real usage that it works with the three of them and add failing tests I will be able to help you.

File: tests/test_sops.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import unittest2
 import mock
+import six
 
 import sops
 
-from sys import version_info
-if version_info.major == 2:
+if six.PY2:
```
Comment (Natim) [inline tests/test_sops.py]: if you don't want to install six, you can also use ``sys.version_info[0] == 2``

**Comment by jvehent** [comment]
Thanks! It still breaks on python3, I'll add tests to help fix the issues.

```bash
$ python3 sops/__init__.py -d example.yaml
Traceback (most recent call last):
  File "sops/__init__.py", line 623, in <module>
    main()
  File "sops/__init__.py", line 199, in main
    write_file(tree, path='/dev/stdout', filetype=otype)
  File "sops/__init__.py", line 592, in write_file
    indent=4))
TypeError: 'str' does not support the buffer interface
```

**Comment by jvehent** [comment]
Almost there! I'm merging this to continue working on it in my branch.

**Pull request merged**